chatkit-core/src/test/kotlin/com/pusher/chatkit/users/state/CurrentUserReceivedTest.kt

```diff
@@ -0,0 +1,60 @@
+package com.pusher.chatkit.users.state
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import com.pusher.chatkit.state.ChatState
+import com.pusher.chatkit.state.CurrentUserReceived
+import com.pusher.chatkit.state.State
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class CurrentUserReceivedTest : Spek({
+
+    val simpleUser = UserInternalType(
+        id = "danielle",
+        createdAt = 1583422281000L,
+        updatedAt = 1583422302000L,
+        name = "daniellevass",
```

Comment:
haven't we agreed to use Alice, Bob etc?

Reply:
👍 i don't recall any conversations (there was one about using Harry Potter names) - have updated for it to say alice.